### PR TITLE
[12.0][MIG] project_task_code

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@pedrobaeza ^ the message above - I have no idea what to do about that

**Comment by alexey-pelykh** [comment]
@rodrigets @cubells I'd like to request your review, if you don't mind

**Comment by pedrobaeza** [comment]
@alexey-pelykh don't worry. It's a missing CLA now that the CLAbot is working properly. We'll handle it internally.

**Comment by pedrobaeza** [comment]
Any way, you are migrating something that is already migrated in #443, so please review that instead.

**Comment by alexey-pelykh** [comment]
Ooops :)

**Comment by alexey-pelykh** [comment]
@pedrobaeza just in case I've updated this one, yet @dbo-odoo proposed an improvement so I've included this as well - not sure though how to refer that piece to @dbo-odoo

**Comment by pedrobaeza** [comment]
I don't get you. What do you mean?

**Comment by alexey-pelykh** [comment]
@pedrobaeza sorry, let me rephrase - I mean that in case that other PR will be unattended for some time, I've updated this one as a backup option

**Comment by pedrobaeza** [comment]
OK, I understand now, but the respectful path should be:

- Take the other branch
- Add your modifications on top of it
- Propose a new PR with that

**Comment by alexey-pelykh** [comment]
Good point! @pedrobaeza there's a #453 to the rescue 👍
